### PR TITLE
docs(perf) update perf test framework API

### PR DESCRIPTION
**Pull request by fffonion**
### Summary
Update perf test framework to latest API

### CODE REVIEW

**Review by mayocream — APPROVED** [review]
LGTM

File: src/gateway/plan-and-deploy/performance-testing-framework.md [hunk 1]
```diff
@@ -302,29 +323,47 @@ Starts to send load to Kong using `wrk`. Throws error if any. Options is a Lua t
...
+
+```lua
+local charts = require "spec.helpers.perf.charts"
+charts.options({
+  suite_sequential = true,
+  xaxis_title = "Upstreams count",
+})
+```
+
+Otherwise, the chart will treat results in same test file as independent tests
```
Comment (cloudjumpercat) [inline src/gateway/plan-and-deploy/performance-testing-framework.md:365]:
```suggestion
If there are no correlations, the chart treats results in same test file as independent tests 
```

Comment (cloudjumpercat) [reply src/gateway/plan-and-deploy/performance-testing-framework.md:365]: @fffonion Is this an accurate revision?

File: src/gateway/plan-and-deploy/performance-testing-framework.md [hunk 2]
```diff
@@ -302,29 +323,47 @@ Starts to send load to Kong using `wrk`. Throws error if any. Options is a Lua t
...
+and draw a bar chart.
```
Comment (cloudjumpercat) [inline src/gateway/plan-and-deploy/performance-testing-framework.md:366]:
```suggestion
and draws a bar chart.
```

File: src/gateway/plan-and-deploy/performance-testing-framework.md [hunk 3]
```diff
@@ -345,7 +384,7 @@ Saves Kong error log to path. Throws error if any.
 Enable charts generation, throws error if any.
 
 Charts are enabled by default; the data of charts are fed by `perf.combine_results`.
-The generated charts are `SVG` files and saved under `output` directory.
+The generated JSON results for creating charts are stored under `output` drectory.
```
Comment (cloudjumpercat) [inline src/gateway/plan-and-deploy/performance-testing-framework.md:387]:
```suggestion
The generated JSON results for creating charts are stored under the `output` directory.
```

File: src/gateway/plan-and-deploy/performance-testing-framework.md [hunk 4]
```diff
@@ -361,6 +400,20 @@ Loads the pgdump from the path and replaces the Kong database with the loaded da
 will also patch a services address to the upstream unless `dont_patch_service` is set to false,
 it must be called after `perf.start_upstream`. Throws error if any.
 
+## Charts
+
+Charts are not rendered by default, there's a reference implementation to draw graphs on all JSON
+data stored in `output` directory. Use the following commands to draw graph:
```
Comment (cloudjumpercat) [inline src/gateway/plan-and-deploy/performance-testing-framework.md:406]:
```suggestion
data stored in the `output` directory. Use the following commands to draw graphs:
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]

**Comment by cloudjumpercat** [comment]
@fffonion When you've added all the content you want to add, could you add the "review: copyedit" label and I'll review the changes? Thanks!

**Comment by cloudjumpercat** [comment]
@fffonion When you have a chance, could you add the "review: copyedit" label if your changes are complete and I'll review the changes? Thanks!

File: src/gateway/plan-and-deploy/performance-testing-framework.md [hunk 5]
```diff
@@ -302,29 +380,47 @@ Starts to send load to Kong using `wrk`. Throws error if any. Options is a Lua t
...
+
+```lua
+local charts = require "spec.helpers.perf.charts"
+charts.options({
+  suite_sequential = true,
+  xaxis_title = "Upstreams count",
+})
+```
+
+Otherwise, the chart will treat results in same test file as independent tests
```
Comment (fffonion) [inline src/gateway/plan-and-deploy/performance-testing-framework.md:422]:
```suggestion
If this option is not set, the chart treats results in same test file as independent tests
```

**Comment by fffonion** [comment]
just did, @cloudjumpercat thanks for the review!

File: src/gateway/reference/performance-testing-framework.md [hunk 6]
```diff
@@ -16,22 +16,67 @@ The framework uses [busted](https://olivinelabs.com/busted/) and some
...
+
+Install Lua development dependencies of Kong being installed and invoke each test file with `busted`. The following runs the simple RPS and latency
+test using `docker` driver.
+
+```shell
+PERF_TEST_USE_DAILY_IMAGE=true PERF_TEST_VERSIONS=git:master,git:perf/your-other-branch bin/busted -o gtest spec/04-perf/01-rps/01-simple_spec.lua
+```
+
+User can also run the performance test framework inside a Kong container from scratch, from other development tools
+like [gojira](https://github.com/kong/gojira) or [pongo](https://github.com/kong/kong-pongo), if they don't have development environment.
```
Comment (cloudjumpercat) [inline src/gateway/reference/performance-testing-framework.md:31]:
```suggestion
like [gojira](https://github.com/kong/gojira) or [Pongo](https://github.com/kong/kong-pongo), if they don't have development environment.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]

**Review by cloudjumpercat — APPROVED** [review]

**Pull request merged**